### ISSUE TITLE
Motivation and Vision of this Package

### ISSUE THREAD

**Issue by andersonaddo** [maxkomarychev/react-native-ultimate-config#12]
Hello!
As I'm sure you surmised, I learnt about this package from your comment in the [maintenance discussion thread of react-native-config](https://github.com/luggit/react-native-config/issues/382).
It looks like, at least for now, this project is probably a safer bet for me to use since you're actively maintaining it. However, I am curious about your vision for this repo.
For example: what motivated you to put in the time to make this? Was it due to that thread I linked? Or something else? Also, do you plan to eventually get this merged into the react-native-community organization, or you plan to maintain this alone for the foreseeable future?

**Comment by maxkomarychev** [comment]
hey,

I had a challenge maintaining a project with > 5 environments and managing all the variables through native projects was a real pain. I found react-native-config >1 year ago but it wasn't in a good shape already (various issues around cocoapods setup and whatnot) and it didn't work well for me. Anyways. The idea behind react-native-config was quite simple so I ended up creating my own package with more capabilities and cleaner (imo) implementation.

regarding community: according to this https://github.com/react-native-community/discussions-and-proposals/issues/176 there is no real consensus on what's the process of adding more packages into community. if my library renders to be useful to others and there will be clear guideline from community on how it can be accepted I may consider transfer.

in general: based on my needs this library has reached feature completeness, but I am open to consider more functionality

**Comment by rizwanCB** [comment]
I personally found it so easy to work with this package! it is super strong!  thankyou for the efforts

**Comment by isilher** [comment]
I found this library after struggling with react-native-config for a day and I must say, what a relief! react-native-ultimate-config worked out of the box and the documentation is very complete and clear. Thank you very much for sharing this project with us! 

I made a proof of concept for a simple white label setup with react-native and react-native-ultimate-config. Feel free to add it to the cookbook if you think it may be useful to anyone. https://github.com/isilher/red-vs-blue
